packages/backend/src/database/migrations/20240219150917_scheduler_active_flag.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable('scheduler', (t) => {
+        t.boolean('active').defaultTo(true);
```

Comment:
alternative implementation: 

use a string or an emum instead of a boolean that we can use for multiple conditions (disabled, send_once....) 

I think is a bit harder to use, I personally prefer to have 2 columns, 1 with the boolean and another with the condition.